Utils/pom.xml

```diff
@@ -323,7 +323,7 @@
             <dependency>
                 <groupId>com.microsoft.azure</groupId>
                 <artifactId>azure-storage</artifactId>
-                <version>4.0.0</version>
+                <version>6.1.0</version>
```

Comment:
Looks like the current stable version is 8.0.x, should we catch up it now?

Reply:
The doc is pointing to 7.0.0 (https://docs.microsoft.com/en-us/java/api/overview/azure/storage?view=azure-java-stable). 

Sure, 6.1.0 does make sense, let's keep it aligned with Azure SDK.